modules/structs/notifications.go

```diff
@@ -21,14 +21,28 @@ type NotificationThread struct {
 
 // NotificationSubject contains the notification subject (Issue/Pull/Commit)
 type NotificationSubject struct {
-	Title            string    `json:"title"`
-	URL              string    `json:"url"`
-	LatestCommentURL string    `json:"latest_comment_url"`
-	Type             string    `json:"type" binding:"In(Issue,Pull,Commit)"`
-	State            StateType `json:"state"`
+	Title            string            `json:"title"`
+	URL              string            `json:"url"`
+	LatestCommentURL string            `json:"latest_comment_url"`
+	Type             NotifySubjectType `json:"type"`
```

Comment:
The binding is still useful?

Reply:
go should serialize it correctly but I'll but it back to throw an error if incorect.

this is an copy/paste issue from the go-sdk, didnt expect to get that lgtm that quic :D 
(code was created while working on the sdk)